package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ember-table",
-  "version": "4.0.0",
+  "version": "5.0.0-beta.0",
```

Comment:
Can we publish this beta to NPM so people can easily try it with their apps? Or are we planning to cut the 5.0 stable version soon?

Reply:
Yeah, I don't see this on public NPM.

Reply:
I have not published anything yet. Mostly, just wanted a review + sanity check on the changes :)